composer.json

```diff
@@ -13,7 +13,9 @@
         "silverstripe/mimevalidator": "2.3.x-dev"
     },
     "require-dev": {
-        "phpunit/phpunit": "^9.5"
+        "phpunit/phpunit": "^9.5",
+        "silverstripe/versioned": "^1",
+        "mikey179/vfsstream": "^1.6.10"
```

Comment:
Are we doing vfsstream via composer.json or via ci? It's currently added explicitly in the ci... should we remove it from the ci and only include it in composer.json for the modules that need it?

Reply:
Perhaps, though we can do that later, there's already an issue for that https://github.com/silverstripe/gha-ci/issues/6